src/pages/user_profile/UserProfile.js

```diff
@@ -27,6 +31,27 @@ export default function UserProfile(props) {
 
   const [notificationData, setNotificationData] = useState({});
 
+  const updateRef = ({key, value}) => {
+    setName(value);
+    let newUser = cloneDeep(user);
+    newUser.displayName = value;
+    setUpdatedUser(newUser);
+  };
+
+  const updateName = () => {
+    if (updatedUser.displayName) {
+      User.update(updatedUser, updatedUser.userId).then((response) => {
```

Comment:
I think we're using the wrong API here. This one calls https://github.com/DataBiosphere/consent/blob/38ea7d8e7273fa724c145a74c9a556fefa751954/src/main/java/org/broadinstitute/consent/http/resources/UserResource.java#L208 which is limited to Admin users. I think we want to call https://github.com/DataBiosphere/consent/blob/38ea7d8e7273fa724c145a74c9a556fefa751954/src/main/java/org/broadinstitute/consent/http/resources/UserResource.java#L229 instead.